cross/fish/Makefile

```diff
@@ -0,0 +1,26 @@
+PKG_NAME = fish
+PKG_FULL_NAME = fish-shell
+PKG_EXT = tar.gz
+PKG_DOWNLOAD_METHOD = git
+PKG_GIT_HASH = 0ddd0ed
+PKG_DIST_SITE = https://github.com/fish-shell/$(PKG_FULL_NAME).git
+PKG_DIST_FILE = $(PKG_NAME)-git$(PKG_GIT_HASH).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-git$(PKG_GIT_HASH)
+
+DEPENDS = cross/ncurses
+
+HOMEPAGE = http://fishshell.com
+COMMENT  = fish is a smart and user-friendly command line shell for OS X, Linux, and the rest of the family.
+LICENSE  = GPLv2
+
+PRE_CONFIGURE_TARGET = myPreConfigure
+GNU_CONFIGURE = 1
+CONFIGURE_ARGS = --prefix=$(INSTALL_DIR)$(INSTALL_PREFIX)
+
+ENV += CXXFLAGS="-I$(INSTALL_DIR)$(INSTALL_PREFIX)/include" LDFLAGS="-L$(INSTALL_DIR)$(INSTALL_PREFIX)/lib"
```

Comment:
Yes, I triple checked with the latest avoton toolkit and spksrc. I think due to the configure script of fish-shell, these includes don't get to the actual build step unless specifically defined. Took me quite a bit to realise =)